example/burrow_integration/burrow_webgrpc.py

```diff
@@ -0,0 +1,109 @@
+address = deploy_contract(contract_bytecode)
+# 2 modify state -> modify's iroha state
+add_coin('wtr#energy','500')
+# 3 query for state using web3grpc -> print result
+web3grpc_bytecode = create_web3grpc_bytecode(address)
+iroha_address = '0.0.0.0:28660'
+iroha_account_sender = '"admin@test"'
+sm_address = '"%s"' % address
+sm_data = '"%s"' %web3grpc_bytecode
+cmd_to_execute = """curl %s -X POST   -H "Content-Type: application/json"   --data '{"jsonrpc":"2.0","method":"eth_call","params":[{"from": %s, "to" :%s, "data": %s }, "latest"],"id":1}' """ % (iroha_address,iroha_account_sender, sm_address, sm_data)
+val = os.popen(cmd_to_execute).read()
+print("val is ",val)
```

Comment:
IMO it is better not to use external command (curl), but to use libraries. You can sens POST with python: https://www.geeksforgeeks.org/get-post-requests-using-python/ or https://www.scrapingbee.com/blog/how-to-send-post-python-requests/